src/Main.ts

```diff
@@ -186,6 +214,10 @@ export class Main {
             controller: {
                 onEvent: async(request) => {
                     const ev = request.getData();
+                    if (this.bridgeBlocker?.isBlocked) {
```

Comment:
We might still want to handle admin commands

Reply:
Excluded in 27aa15a